actions/java-maven-release/README.md

```diff
@@ -0,0 +1,31 @@
+# java-maven-release
+
+This action releases Java Maven artifacts by createing a tag on GitHub.
+
+## Input Parameters
+
+| Name              | Required | Default Value |  Type  | Description                                                                                        |
+| ----------------- | :------: | :-----------: | :----: | -------------------------------------------------------------------------------------------------- |
```

Comment:
Probably already missing where you copied it from, but `release-type` is not documented as an input